### ISSUE TITLE
[Feature request] enable user to pick language

### ISSUE THREAD

**Issue by dontinelli** [home-assistant/intents#1013]
According to my understanding of the documentation, currently the language used for Conversation is the one set in the properties. In my view it should be possible a different language in the configuration file. Even better would be, if each user could pick the preferred language.

**Comment by TheFes** [comment]
Each user can select their own language, because the language set in the user profile is used.

**Comment by dontinelli** [comment]
> Each user can select their own language, because the language set in the user profile is used.

Good to know. I thought that because the language setting is under general settings, it is applied globally. Nevertheless it would be helpful if a different language could be picked. In fact, I do like HA to be displayed in German but would prefer Assist to talk Swiss German.

**Comment by TheFes** [comment]
I agree with that. I like my HA to be in English, but would like use assist in Dutch.
So for now my HA is in Dutch as well..

**Comment by Arkkimaagi** [comment]
Yep, I would prefer my HA to be English and assist in Finnish. I have the setup with my iPhone, all UI is English, but Siri talks Finnish. 

My reasoning for both HA and iPhone UI vs spoken content:
* Translations from English to Finnish for technical things are usually not that great (GUI in English)
* If I run into a problem, I prefer to be able to google for a solution in English as there is more content to search online (GUI in English)
* I pronounce better Finnish than English (Spoken UI in Finnish)
* Siri and Assist deal with a more humane everyday things where naming is more standard (compared to obscure network settings I prefer to handle over GUI) (Spoken UI in Finnish)
